modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/index.ts

```diff
@@ -91,6 +91,19 @@ export interface IBaseActions {
 	itemId: number | string;
 }
 
+export interface ICardsView {
+	items: Array<any>;
+	schema: ICardSchema;
+}
+
+export interface ICardSchema {
+	description: string;
+	image: string;
+	link: string;
+	symbol: string;
+	title: string;
+}
+
```

Comment:
Are these interfaces used anywhere else besides `Cards.tsx`? If no, we can keep them internal to the file, without exporting.

Reply:
For the sake of this PR, only `ICardSchema` are needed, and inside `Cards.tsx`